arm-hdinsight/2015-03-01-preview/swagger/cluster.json

```diff
@@ -404,6 +404,12 @@
                 },
                 "configurations": {
                     "type": "object",
+                    "additionalProperties": {

```

Comment:
This will change the type of `configurations` in C# from 
```chsarp
Object Configurations
```
to 
```csharp
Dictionary<string, Dictionary<string, string> configurations
```